main.go

```diff
@@ -41,4 +42,12 @@ func main() {
 	if err != nil {
 		log.Fatal("Could not start GlusterD Rest Server. Aborting.")
 	}
+
+	// Starting etcd daemon upon starting of GlusterD
+	etcd_start := exec.Command("/bin/etcd")
```

Comment:
Have you tested these changes? I don't think etcd instance will come up here. OTOH, we'd need to pass some additional parameters to start etcd. Refer to etcd documentation.